src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/ChannelsController.cs

```diff
@@ -40,7 +40,11 @@ public IActionResult Get(string classification = null)
                 query = query.Where(c => c.Classification == classification);
             }
 
-            List<Channel> results = query.AsEnumerable().Select(c => new Channel(c)).ToList();
+            List<Channel> results = query.Include(ch => ch.ChannelReleasePipelines)
```

Comment:
Please don't include all this information in this API. This is a "list" api, it shouldn't grab all the data for each item.

Reply:
You mean only remove it on this specific method, right?

Reply:
yea, the apis that return a single channel can include it. This one should not.